services/authentication/src/main/java/entity/UserStore.java

```diff
@@ -0,0 +1,26 @@
+package entity;
+
+/**
+ * Created by Michele on 12/07/2017.
+ */
+
+/**
+ * @author Michele Donati
+ * This inteface models the <em>Users' database domain</em>.
+ */
+
+public interface UserStore {
+
+    /**
+     * Maps an <em>User</em> object in a JSon object.
+     */
+    public void mapUsersToJSon();
+
+    /**
+     *
+     * @param username Identifies the <em>User</em>'s <em>username</em> that must be authenticated.
```

Comment:
The words after @param are the names of the parameters in the method's parameters list.